src/library_wasmfs_node.js

```diff
@@ -67,10 +67,10 @@ mergeInto(LibraryManager.library, {
   _wasmfs_node_get_mode: function(path_p, mode_p) {
     let stat = wasmfsNodeLstat(UTF8ToString(path_p));
     if (stat === undefined) {
-      return 0;
+      return 1;
     }
-    {{{ makeSetValue('mode_p', 0, 'stat.mode', 'i32') }}}
-    return 1;
+    {{{ makeSetValue('mode_p', 0, 'stat.mode', 'i32') }}};
+    // return 0;
```

Comment:
```suggestion
    return 0;
```

Reply:
Should we replace all the `// return 0;` in this file with actual returns? I've been depending on the implicit return of undefined converting to 0 in Wasm.

Reply:
It is ok to do so, but in that case maybe the comment should be more explicit. I see we have this comment already in library.js:
```js
    // implicit 0 return to save code size (caller will cast "undefined" into 0
    // anyhow)
```
Perhaps `// implicit return 0` or `// return 0 implicitly`?